### DISCUSSION TITLE
Wallet password

### DISCUSSION THREAD

**Discussion by Nikita036** [iotaledger/firefly#2212] [Help & Support] [answered]
Hello! I forgot the password for the Firefly purse, I only know the pincode. 24 there are no words. there is a backup copy. I can prove that this is my wallet by transactions with Binance. help please

**Comment by ScaNicky** [comment, marked as answer]
Without the stronghold password or 24 words seed it is impossible to access your funds.

**Reply by Nikita036** [reply]
Разработчики не могут помочь?

**Comment by Nikita036** [comment]
у меня есть stronghold, но нет пароля24.01.2022, 21:00, "ScaNickyIsWatchingYou" ***@***.***>:
Without the stronghold password or 24 words seed it is impossible to access your funds.

—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: ***@***.***>
-- С уважением, Никита Ткачев+79507739296

**Comment by Nikita036** [comment]
у меня есть stronghold но нет пароля  24.01.2022, 21:00, "ScaNickyIsWatchingYou" ***@***.***>: Without the stronghold password or 24 words seed it is impossible to access your funds.—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.You are receiving this because you authored the thread.Message ID: ***@***.***>  -- С уважением,Никита Ткачев+79507739296
